.github/ISSUE_TEMPLATE/spec_request.md

```diff
@@ -0,0 +1,113 @@
+---
+title: 'Contribute around a new format specification (newbie edition)'
+labels: spec-request
+---
+<!--
+Wanna 
+* ... find collaborators to work on the format?
+* ... inform the world that you have started working on the spec for a format to avoid effort being wasted?
+* ... discuss implementation details?
+* ... just dump info useful for implementing the spec?
+
+Then this template is for you.
+
+Since for every format we deal with the same info and since it is beneficial for both human and bot (one is planned!) readers to have the info encoded into a messages with the same schema, we have designed the format for this kind of issues. Please, follow it.
+
+A request for a format specification consists of some YAML blocks followed by a free-form text description. Each YAML block has own purpose, its purpose is described in the comment within that block. 
+
+We expect you to:
+* fill in all the fields you can and want to fill in;
+* delete all the unfilled fields;
+* delete all the comments (both YAML and HTML) from this template (including this one!).
+* make sure that YAML blocks contain well-formed YAML;
+* make sure that blocks contents do not violate their schemas;
+* make sure that all the fields marked as MANDATORY are filled in.
+
+-->
+```yaml
+# The purposes of this block are:
+# * storing the info uniquily identifying the format to make it searcheable;
+# * storing useful information about the format in a convenient structured way;
+# * providing an implementer with a boilerplate which can be just copied into a KSY file;
+# The content of this block must be a part of a valid KSY.
+# This block must be present.
+
+meta: # see https://doc.kaitai.io/ksy_reference.html#meta for more info
+  id: # insert an ID (matches the regexp [a-z][\da-z_]*[\da-z]) you feel right to assign to the format here. KS currently has no namespaces, so the id MUST be unique within all the repo and you should be sure there will be no collisions. If in doubt, add a prefix of an app name. MANDATORY.
```

Comment:
I don't think a regex pattern will help newcomers very much. Maybe just give some examples of valid `id`s instead and/or link to the relevant docs.

Reply:
I assumme that every programmer (this tool is for programmers) knows regexps.

Reply:
You can do a lot of programming without having to use regexes, especially in languages that have good standard string operations. Also even if you use regexes, you don't necessarily know them well enough to understand a pattern like `[a-z][\da-z_]*[\da-z]` without having to stop and think about it for a few seconds.

I'd be fine with putting this pattern somewhere else (for example [ksy_schema](https://github.com/kaitai-io/ksy_schema)), but in this case it should be enough to write something like "IDs use `snake_case`".